lib/atscppapi/src/include/atscppapi/Transaction.h

```diff
@@ -30,7 +30,7 @@
 #include "atscppapi/shared_ptr.h"
 #include "atscppapi/ClientRequest.h"
 #include "atscppapi/Response.h"
-
+#include <../ts/apidefs.h>
```

Comment:
We can't do this. The pattern we've always followed with the ats c++ api is that it shouldn't require including any C header files. We've typically just duplicated enums so TSOverridableConfigKey should be an enum in Transaction.